tests/cupyx_tests/scipy_tests/special_tests/test_logsumexp.py

```diff
@@ -0,0 +1,108 @@
+import pytest
+
+import numpy
+
+import cupy
+from cupy import testing
+import cupyx.scipy.special  # NOQA
+
+
+@testing.with_requires('scipy')
+class TestLogsumexp:
+
+    @testing.for_all_dtypes(no_bool=True)
+    @testing.numpy_cupy_array_equal(scipy_name='scp')
```

Comment:
Please use `numpy_cupy_allclose` to compare floating point numbers

Reply:
Hi, updated all. Thanks!